tests/instances.spec.ts

```diff
@@ -398,3 +398,15 @@ test("Migrate instance root storage volume to a different pool", async ({
   await migrateInstanceRootStorage(page, instance, "default", serverClustered);
   await deletePool(page, targetPool);
 });
+
+test("'Other' tab is removed when config/creating Instances, on LXD Version 5.0", async ({
```

Comment:
This test is failing, and likely the other one will fail too. If you look at the playwright [video](https://canonical.github.io/lxd-ui/reports/pr-944/11241353075/1/#?testId=b980ab2a78df8d904d9c-9be92e23cd8b98d2fb66) for this failing test, you can see the "Other" tab is hidden, so likely `page.getByText("Other")` is resolving to some other components on the page.

To develop these tests locally, you have two options:
1. revert lxd to 5.0 and test
2. emulate the condition by just hiding the "Other" tab without using api extension, make this test pass locally without the `skip`, then it should pass in CI as well.

I personally prefer approach 2, since reverting to lxd 5.0 will require you to uninstall lxd and reinstall 5.0 plus doing all of the initial configs.